drizzlepac/hlautils/se_source_generation.py

```diff
@@ -387,46 +412,119 @@ def write_catalog(seg_cat, catalog_filename, product="tdp"):
         dd = Column(dec_icrs, name="Dec_icrs")
```

Comment:
From the astropy docs, it appears that you should be able to define the 'description', 'units' and even 'format' when you define Column and have those values get added to the output table metadata automatically.  This may eliminate the need to add the .description and .unit lines for these columns from below... if it actually works.